dev/com.ibm.ws.microprofile.reactive.streams.operators.cdi/bnd.bnd

```diff
@@ -53,10 +53,10 @@ testsrc: test/src
 	com.ibm.websphere.javaee.cdi.1.2;version=latest,\
```

Comment:
i think we agreed that only cdi-2.0 support was required??

Reply:
Edit done.